README.md

```diff
@@ -48,61 +38,15 @@ networking or storage code.
 ### Installation
 
 ```sh
-npm install boardgame.io
+npm install @think-and-dev/cartesi-boardgame
 ```
 
-### Documentation
+### Documentation's boardgame.io
 
 Read our [Full Documentation](https://boardgame.io/documentation/) to learn how to
 use boardgame.io, and join the [community on gitter](https://gitter.im/boardgame-io/General)
 to ask your questions!
 
```

Comment:
Also, a comment that makes it clear that this is an early beta version and does not have all of the features yet might be a good addition.